src/Confluent.Kafka/project.json

```diff
@@ -14,8 +14,7 @@
     },
 
     "frameworks": {
-        "net451": { },
```

Comment:
I just realized the reason I use that script is mostly VS Code / omni-sharp which can't deal with net451 being in there (by default.. and if there's a config setting I didn't find it).